ec2net-functions

```diff
@@ -18,15 +18,25 @@ if [ -z "${INTERFACE}" ]; then
   exit
 fi
 
-HWADDR=$(cat /sys/class/net/${INTERFACE}/address 2>/dev/null)
-while test "${HWADDR}" = "00:00:00:00:00:00"; do
-  sleep 0.1
-  HWADDR=$(cat /sys/class/net/${INTERFACE}/address 2>/dev/null)
-done
+# Support alternate locations for config files in order to facilitate
+# testing.
+ETCDIR=${ETCDIR:-/etc}
```

Comment:
should probably also have a variable for `/sys` ?

Reply:
At the moment, it wouldn't be used for anything, since we wrap the only `/sys` access in a conditional block that we avoid when running tests. But redirecting `/sys` from somewhere else might be a better approach...

Reply:
I've changed things a bit in the latest revision such that we do provide /sys, and we use our custom /sys tree to provide the `HWADDR` value in ec2net-functions.  This results in tests looking a bit more like reality.